src/modules/BuilderPlace/actions/builderPlace.ts

```diff
@@ -574,6 +605,11 @@ export const addBuilderPlaceCollaborator = async (body: AddBuilderPlaceCollabora
 export const updateBuilderPlace = async (builderPlace: UpdateBuilderPlace) => {
   let errorMessage;
   try {
+
+    if (builderPlace.discordWebhook != "" && builderPlace.discordWebhook != undefined) {
```

Comment:
2 things here: 
- both conditon are falsy, so just do: if (builderPlace.discordWebhook) 
- it's better to move this check logical inside your function, else you have to duplicate it everywhere we need to use it